Controller/InputProcessor.cs

```diff
@@ -1,7 +1,67 @@
+using HashPhotoSlideshow.Model;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Text;
+
 namespace HashPhotoSlideshow.Controller
 {
     public class InputProcessor
     {
-        //first commit
+        public List<Photo> _photo;
+
+        public InputProcessor()
+        {
```

Comment:
No need for constructor. All functionality can go in the static ReadInputFile function.